### DISCUSSION TITLE
render m2a field inline instead of single line preview

### DISCUSSION THREAD

**Discussion by marcusand** [directus/directus#13209] [Feature Requests]
Hello :)

I have a question related to that discussion: https://github.com/directus/directus/discussions/12384 

Are there any efforts for adding something like that to the directus core?
I would love a m2a field that displays all related items inline instead of just the single line preview that 
you have to select in order to see and edit the actual content.

Especially for editing article style content that would be super helpful, I think. 

Thx!

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
